### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by knowbody**
refactored some of the code, check if you like it @kenwheeler

### CODE REVIEW

File: src/appear.js [hunk 1]
```diff
@@ -21,10 +21,10 @@ const Appear = React.createClass({
     };
   },
   componentWillMount() {
-    this.context.export || this.context.flux.stores.SlideStore.listen(this._storeChange);
+    return this.context.export || this.context.flux.stores.SlideStore.listen(this._storeChange);
```
Comment (kenwheeler) [inline src/appear.js]: why retrun

Comment (knowbody) [inline src/appear.js]: there must be some return in those functions, else eslint is complaining

**Comment by kenwheeler** [comment]
Ok so everything looks pretty good, except for build. Build should wrap everything up into dist. HTML & images included. @knowbody

**Comment by knowbody** [comment]
@kenwheeler forgot about that, will fix now. thanks

**Pull request merged**